resources/js/app.js

```diff
@@ -62,11 +62,11 @@
 				$scope.projects_url_dict = {}
 				
 				angular.forEach($scope.projectList, function(value, key){
-					value["url"] = value["name"].split(' ').join('_').toLowerCase();
+					value["url"] = encodeURIComponent(value["name"].split(' ').join('_').toLowerCase());
```

Comment:
Yeah actually it replaced spaces with %20, so links won't be easy to type directly, like improve%20diff%20handling, after split/join its like improve_diff_handling